ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -163,6 +164,10 @@ ORK_CLASS_AVAILABLE
                                                         maximumValueDescription:(nullable NSString *)maximumValueDescription
                                                         minimumValueDescription:(nullable NSString *)minimumValueDescription;
 
++ (ORKTextScaleAnswerFormat *)textScaleAnswerFormatWithTextChoices:(NSArray <ORKTextChoice *> *)textChoices
+                                                      defaultIndex:(NSInteger)defaultIndex
```

Comment:
Decided to stay with NSInteger because other scale sliders use it also.
Don't want one slider to have to use NSIntegerMax and another NSUIntegerMax.
Seems a bit inconsistent.
Did update the double to NSInteger though! :+1: